embark-ui/src/components/DebugButton.js

```diff
@@ -29,10 +30,12 @@ class DebugButton extends React.Component {
 }
 
 DebugButton.defaultProps = {
-  forceDebuggable: false
+  forceDebuggable: false,
+  onClick: () => undefined
```

Comment:
Wouldn't that create an issue if it is undefined? `undefined is not a function` and such?